### PR TITLE
Extract format() from format_and_push()

### LINKED ISSUE #66: Separate format and push functions

### PR DESCRIPTION
**Pull request by EFT-Defra**
Closes #66 by extracting ``format()`` from ``format_and_push()``.

### CODE REVIEW

File: src/oiflib/core.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 """Core functions for oiflib."""
 
 from oiflib.extract import extract  # noqa: F401
+from oiflib.format import format  # noqa: F401
```
Comment (EFT-Defra) [inline src/oiflib/core.py:4]: Make ``format()`` available from ``oiflib.core``

File: src/oiflib/format.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            value_column: "Value",
+        }
+    else:
+        columns = {
+            year_column: "Year",
+            disaggregation_column: disaggregation_column_new,
+            value_column: "Value",
+        }
+
+    return df[columns.keys()].rename(columns=columns)
```
Comment (EFT-Defra) [inline src/oiflib/format.py:86]: Define ``format()``

File: tests/conftest.py [hunk 3]
```diff
@@ -194,3 +194,74 @@ def schema_dict(
...
+@fixture
+def expected_air_one_formatted_rename_disaggregation_column() -> DataFrame:
+    """Example of formatted Air One DataFrame."""
+    return DataFrame(
+        data={
+            "Year": [1990] * 5 + [1991] * 5,
+            "Pollutant": ["NH3", "NOx", "SO2", "NMVOC", "PM2.5"] * 2,
+            "Value": [2 / 2 * 100] * 5 + [3 / 2 * 100] * 5,
+        },
+    )
```
Comment (EFT-Defra) [inline tests/conftest.py:267]: Define fixtures to test ``format()``.

File: tests/test_format.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        df=expected_air_one_enriched,
+        year_column="EmissionYear",
+        value_column="Index",
+        **additional_args,
+    )
+
+    assert_frame_equal(
+        left=returned,
+        right=expected,
+    )
```
Comment (EFT-Defra) [inline tests/test_format.py:43]: Define tests for ``format()``.

**Review by Jtsmith275 — APPROVED** [review]
Hi Ed this looks like a sensible approach

**Pull request merged**